portable/GCC/ARM7_AT91SAM7S/lib_AT91SAM7X256.h

```diff
@@ -1,1469 +1,1396 @@
-//* \brief Output Enable PIO
-//*----------------------------------------------------------------------------
-__inline void AT91F_PIO_OutputEnable(
-        AT91PS_PIO pPio,   // \arg  pointer to a PIO controller
-        unsigned int flag) // \arg  pio output to be enabled
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_IsSet */
+/** \brief Test if PIO is Set */
+/**---------------------------------------------------------------------------- */
+    __inline int AT91F_PIO_IsSet( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                  unsigned int flag ) /* \arg  flag to be tested */
+    {
+        return( AT91F_PIO_GetStatus( pPio ) & flag );
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_OutputEnable */
+/** \brief Output Enable PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_OutputEnable( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                          unsigned int flag ) /* \arg  pio output to be enabled */
+    {
         pPio->PIO_OER = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_OutputDisable
-//* \brief Output Enable PIO
-//*----------------------------------------------------------------------------
-__inline void AT91F_PIO_OutputDisable(
-        AT91PS_PIO pPio,   // \arg  pointer to a PIO controller
-        unsigned int flag) // \arg  pio output to be disabled
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_OutputDisable */
+/** \brief Output Enable PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_OutputDisable( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                           unsigned int flag ) /* \arg  pio output to be disabled */
+    {
         pPio->PIO_ODR = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_GetOutputStatus
-//* \brief Return PIO Output Status
-//*----------------------------------------------------------------------------
-__inline unsigned int AT91F_PIO_GetOutputStatus( // \return PIO Output Status
-        AT91PS_PIO pPio) // \arg  pointer to a PIO controller
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_GetOutputStatus */
+/** \brief Return PIO Output Status */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PIO_GetOutputStatus( /* \return PIO Output Status */
+        AT91PS_PIO pPio )                            /* \arg  pointer to a PIO controller */
```

Comment:
Does this need to be on multiple lines?

Reply:
It doesn't but the existing format has it on multiple lines. This is just swapping their C++ style comment out for a C style one. Can change this if desired by others.

Reply:
It is just inconsistent is what I noticed, some are all one one line, others are spread across multiple, and some have just the closing bracket on a new line